docs/blog/2023-07-18-State-of-Affairs-in-OKD-CI-CD.md

```diff
@@ -0,0 +1,99 @@
+dependency on OpenShift, an remnant from the past when its Origin predecessor was a downstream rebuild of OpenShift 3,
+and concludes with an outlook on how OKD Streams will help users, developers and partners to experiment with future
+OpenShift.
+
+## Fedora CoreOS and CentOS Stream CoreOS
+
+[Fedora CoreOS is built with a Jenkins pipeline running in Fedora's infrastructure](
+https://github.com/coreos/fedora-coreos-pipeline) and is being maintained by the Fedora CoreOS team.
+
+CentOS Stream CoreOS is built with a [Tekton pipeline](https://github.com/okd-project/okd-coreos-pipeline/) running in a
+OpenShift cluster on [MOC](https://massopen.cloud/)'s infrastructure and pushed to `quay.io/okd/centos-stream-coreos-9`.
+The SCOS build pipeline is owned and maintained by the OpenShift OKD Streams team and [SCOS builds are being imported
+from `quay.io` into OpenShift CI as `ImageStream`s](
+https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/okd/scos.yaml).
+
+## OpenShift payload components
+
+At the time of writing, most payload components for OKD/FCOS and OKD/SCOS get mirrored from OCP CI releases. OpenShift
+CI ([Prow](https://docs.prow.k8s.io) and [ci-operator](https://docs.ci.openshift.org/docs/architecture/ci-operator/))
+periodically builds OCP images, e.g. for [OVN-Kubernetes](
+https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ovn-kubernetes/).
+[OpenShift's `release-controller`](https://github.com/openshift/release/tree/master/core-services/release-controller)
+detects changes to image streams, caused by recently built images, then builds and tests a OCP release image. When such
+an release image passes all non-optional tests (also see [release gating](
+https://docs.ci.openshift.org/docs/architecture/release-gating/) docs), the release image and other payload components
+are mirrored to `origin` namespaces on `quay.io`. [For example, at most every 3 hours a OCP 4.14 release image](
+https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.14-ci.json)
+will be deployed (and upgraded) on AWS and GCP and afterwards tested with [OpenShift's conformance test suite](
+https://github.com/openshift/origin). [When it passes the non-optional tests the release image and its dependencies will
+be mirrored to `quay.io/origin` (except for `rhel-coreos*`, `*-installer` and some other images)](
+https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.14-ci.json).
+These OCP CI releases are listed with a `ci` tag at [amd64.ocp.releases.ci.openshift.org](
+https://amd64.ocp.releases.ci.openshift.org/). Builds and promotions of `nightly` and `stable` OCP releases are handled
+differently (i.e. outside of Prow) by the [Automated Release Tooling (ART)](
+https://source.redhat.com/groups/public/openshift/openshift_wiki/openshift_automated_release_tooling_art_team_faqs)
+team.
```

Comment:
I have filed an RFE with the testplatform team to change this flow, and promote images directly to OKD without gating them on OCP CI releases: https://issues.redhat.com/browse/DPTP-3565

It might make sense to mention and link to that here.

Reply:
Ok, I have added a remark about release gating in braces. Better?